src/sync_drive.py

```diff
@@ -181,12 +184,18 @@ def process_file(item, destination_path, filters, files):
     if not wanted_file(filters=filters, file_path=local_file):
         return False
     files.add(local_file)
-    if is_package(item=item):
+    item_is_package = is_package(item=item)
+    if item_is_package:
         if package_exists(item=item, local_package_path=local_file):
+            for f in Path(local_file).glob("**/*"):
```

Comment:
If package already exists we have to add files to list from package, not only package name.